src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +92,10 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(x -> x.getAge() >= fromAge && (x.getSex() == People.Sex.MAN
+                                        ? x.getAge() <= maleToAge : x.getAge() <= femaleToAge))
```

Comment:
```suggestion
                .filter(x -> x.getAge() >= fromAge 
                                        && (x.getSex() == People.Sex.MAN
                                        ? x.getAge() <= maleToAge 
                                        : x.getAge() <= femaleToAge))
```
it is a bit easier to read, what do you think?

Reply:
totally agree

Reply:
done